quinn-udp/src/unix.rs

```diff
@@ -792,7 +792,13 @@ mod gso {
         // #define UDP_MAX_SEGMENTS        (1 << 6UL)
         match set_socket_option(&socket, libc::SOL_UDP, UDP_SEGMENT, GSO_SIZE) {
             Ok(()) => 64,
-            Err(_) => 1,
+            Err(e) => {
+                crate::log::debug!(
+                    "Failed to set `UDP_SEGMENT` socket option ({e}); setting `max_gso_segments = 1`"
```

Comment:
Some applications might create UDP sockets quite frequently (e.g. DNS clients). I think logging this on `info` or higher might be a bit spammy, esp. because it very likely happens again for every socket.

Reply:
Fair enough.

Reply:
We could compromise on this by performing the check at most once per process via a static or similar, but I don't really have a strong opinion.